README.md

```diff
@@ -98,9 +98,10 @@ When an older Jenkins version is used, then the matching older version of the pl
 | bom-2.401.x | 2745.vc7b_fe4c876fa_  |                              |
 | bom-2.414.x | 2982.vdce2153031a_0   |                              |
 | bom-2.426.x | 3208.vb_21177d4b_cd9  |                              |
+| bom-2.440.x | 3413.v0d896b_76a_30d  | Last LTS to support Java 11  |
```

Comment:
2.452.x and 2.462.x still support Java 11 so I am confused by this statement.

Reply:
Thanks for detecting that!  My enthusiasm got the best of me and I made a false statement in that comment.

Fixed in 5092977c00577d369b26ae999257306af9e8d569